projects/demo/src/modules/app/pages.ts

```diff
@@ -42,6 +42,12 @@ export const pages: TuiDocPages = [
         section: $localize`Documentation`,
         title: $localize`Testing`,
         subPages: [
+            {
+                section: $localize`Jest`,
```

Comment:
I have found error (why e2e fails). It here :)
```suggestion
                section: $localize`Documentation`,
```


My mistake here, sorry. It is useless. Could you revert this line
https://github.com/Tinkoff/taiga-ui/pull/2432#issuecomment-1223682621

Reply:
Or we should rename all sections inside this array to `Testing` (and dont revert this comment https://github.com/Tinkoff/taiga-ui/pull/2432#issuecomment-1223682621) 🤔

Reply:
I think **semantically** the second approach (rename all sections inside this array to `Testing`) is better.

@splincode are you agree ?